package.json

```diff
@@ -54,10 +54,8 @@
 		"lint": "npm run syncpack:deps && npm run syncpack:versions && npm run prettier:root && npm run ci:eslint",
 		"lint:fix": "npm run prettier:root:fix && pnpm run -r --no-sort --stream lint:fix",
 		"policy-check": "flub check policy --exclusions build-tools/packages/build-tools/data/exclusions.json",
-		"policy-check-help": "echo Detect (and error on) policy-check violations, like package.json sorting, copyright headers etc. Excludes assert-short-code. Run the check or \"pnpm flub check policy --listHandlers\" for a full list.",
```

Comment:
why was this deleted. I wrote it for a reason. Did we have a bunch of merges just losing changes to package.json files when version updates happened?

Reply:
This is very concerning. I think you're probably right that this was due to a merge resolution gone wrong. But how to find them and prevent them? Not sure.